.github/actions/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "cuttle-actions",
```

Comment:
We need a separate `package.json` here in order to install the `@actions` libs we need without muddying the normal cuttle `package.json`.

Reply:
Is that considered a best practice? It's an interesting idea, I've just never heard of it